code/clue.js

```diff
@@ -6,54 +6,271 @@ const mrGreen = {
+  weight: 12,
+  location: 'Billiard Room'
+}
+
+const poison = {
+  name: 'poison',
+  id: 'poison',
+  weight: 1.5,
+  location: 'Conservatory'
+}
+
+const axe = {
+  name: 'axe',
+  id: 'axe',
+  weight: 8,
+  location: 'Patio'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const bat = {
+  name: 'bat',
+  id: 'bat',
+  weight: 18,
+  location: 'Hall'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const trophy = {
+  name: 'trophy',
+  id: 'trophy',
+  weight: 17,
+  location: 'Lounge'
+}
+
+const pistol = {
+  name: 'pistol',
+  id: 'pistol',
+  weight: 1,
+  location: 'Study'
+}
 
 const suspects = [
   mrGreen,
+  mrMustard,
+  mrsPeacock,
+  mrPlum,
+  msCassandra,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = [
+  'Dining Room', 
+  'Conservatory', 
+  'Kitchen',
+  'Study', 
+  'Library', 
+  'Billiard Room', 
+  'Lounge',
+  'Ballroom', 
+  'Hall', 
+  'Spa', 
+  'Living Room', 
+  'Observatory', 
+  'Theater', 
+  'Guest House', 
+  'Patio',
+]
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
+const randomSelector = (array) => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: ''
+}
+
+const shuffleFavouriteWeapon = () => {
+  suspects.forEach((suspect) => {
+    const randomWeapon = randomSelector(weapons)
+    suspect.favouriteWeapon = randomWeapon.id;
+  })
+}
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+  document.querySelector('.killer-deck').classList.add('shakeX');
+  
+  setTimeout(function() {
+    document.querySelector('.killer-deck').classList.add('bounce');
+  }, 1000);
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  setTimeout(function() {
+    document.querySelector('.killer-deck').classList.remove('shakeX');
+    document.querySelector('.killer-deck').classList.remove('bounce');
+    document.querySelector('.killer-icon').classList.add('hide');
+
+    shuffleFavouriteWeapon();
+    mystery.killer = randomSelector(suspects)
```

Comment:
You seem to use semicolons only from time to time - we have to be consequent about that